virttest/qemu_storage.py

```diff
@@ -631,6 +634,23 @@ def amend(self, params, cache_mode=None, ignore_status=False):
         cmd_result.stderr = results_stderr_52lts(cmd_result)
         return cmd_result
 
+    def resize(self, size, shrink=False, ignore_status=False):
+        """
+        Qemu image resize wrapper.
+
+        :param size: string of size representations.(eg. +1G, -1k, 1T)
+        :param shrink: boolean
+        :param ignore_status: whether to raise an exception when command
+                              returns =! 0 (False), or not (True)
```

Comment:
`=!/!=`?

Reply:
Should have a `:return` in the doc string.

Reply:
updated, thanks

Reply:
any other comments?